action.yml

```diff
@@ -1,4 +1,4 @@
-name: 'Setup Java JDK'
+nnname: 'Setup Java JDK'
```

Comment:
Could you please revert this changes.

Reply:
@dmitry-shibanov  reverted the changes. Thank you.